lib/webhooks.js

```diff
@@ -35,18 +35,24 @@ function makeJob(project, config) {
 
 function startFromCommit(project, payload, send) {
   const config = pushJob(payload);
-  const lastCommit = payload.commits[payload.commits.length - 1];
+  if (payload.commits) {
+    const lastCommit = payload.commits[payload.commits.length - 1];
 
-  if(lastCommit.message.indexOf('[skip ci]') > -1) {
-    return {skipCi: true};
+    if(lastCommit.message.indexOf('[skip ci]') > -1) {
+      return {skipCi: true};
+    }
+  } else if (payload.after) {
+    if (/^0+$/.test(payload.after)) {
```

Comment:
What is that regex testing? Can you leave a comment there for clarity?

Reply:
okay